### ISSUE TITLE
Easy Joe 3: Cannot click main menu buttons

### ISSUE THREAD

**Issue by n0samu** [ruffle-rs/ruffle#9682]
### Describe the bug

Link to SWF: https://i.notdoppler.com/files/easyjoe3.swf

In [Easy Joe 3](http://www.notdoppler.com/easyjoe3.php), clicking any of the main menu buttons does nothing.

### Expected behavior

Clicking Start Game should start the game, clicking the sponsor link should open the sponsor's site, etc.

### Affected platform

Browser's extension

### Operating system

Windows 10

### Browser

Microsoft Edge 110

### Additional information

The game makes some requests to URLs that no longer exist, but the game still works in a Flash-enabled browser at the link provided.

A few different errors are shown in the console, but none are clearly the source of the problem:
```
ERROR core/src/display_object/movie_clip.rs:2405 Error occured when running AVM2 frame script: RustError("Attempted to call \"\" with 1 arguments (more than 0 is prohibited)")
ERROR core/src/avm2/activation.rs:942 TypeError: Error #1009: Cannot access a property or method of a null object reference. (accessing field: com.google.analytics.v4:GoogleAnalyticsAPI::trackPageview)
WARN core/src/stub.rs:126 Encountered stub: AVM2 flash.utils.describeType()
```

**Comment by n0samu** [comment]
Unescape is implemented now, but the Continue button is still unclickable. This seems like a mouse pick issue.

**Comment by n0samu** [comment]
Fixed by #9995
